keps/sig-node/20190129-hugepages.md

```diff
@@ -552,6 +556,18 @@ locality guarantees as a feature of QoS.  In particular, pods in the
 - E2E testing validating its usage.
 -- https://k8s-testgrid.appspot.com/sig-node-kubelet#node-kubelet-serial&include-filter-by-regex=Feature%3AHugePages
 
+## Test Plan
+
+- A test plan will consist of the following tests
+  - Unit tests
+    - Each unit test for enhancement will be implemented.
+  - E2E tests
+    - There is a test suit for huge pages in e2e test, it will be extended to validate enhancements.
+    - here: https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/hugepages_test.go
```

Comment:
How will it be extended?
How much work do you think will have to be done to get those e2e test to validate the changes in this kep?
Are there any open PRs?

Reply:
My team member opened PR to update e2etest only for `container isolation of hugepages`.
https://github.com/kubernetes/kubernetes/pull/87118
(In the PR, we added a new test case on hugepages test suit.)

So, I think another PR should be opened for `supporting multiple size of hugepages` by these guys(@bart0sh @odinuge)

That(need for another PR) is the reason why I added e2etest ont the future plan:
```
### Version 1.19[TBD]

Extending of huge pages test suit of E2E tests and cri-tools for enhancements after GA.
```

Reply:
https://github.com/kubernetes/kubernetes/pull/87118
In the e2e test, a new test case sets a hugepage limit before creating a container, and then when container created, it checks container has the limit as given.
(test case takes given hugepages limit numbers from CRI runtime logs. It is the same mechanism as to how CPU Manager e2e test works.)